akka-cluster-typed/src/test/scala/akka/cluster/typed/ActorRefIgnoreSerializationSpec.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.typed
+
+import akka.actor.typed.{ ActorRef, ActorRefResolver, ActorSystem }
+import akka.actor.typed.scaladsl.Behaviors
+import com.typesafe.config.ConfigFactory
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class ActorRefIgnoreSerializationSpec extends AnyWordSpec with ScalaFutures with Matchers {
+
+  "ActorRef.ignore" should {
+
+    "return  a serializable ActorRef that can be sent between two ActorSystems using remote" in {
+
+      val config = ConfigFactory.parseString(s"""
+        akka {
+          loglevel = debug
+          actor.provider = cluster
+          remote.classic.netty.tcp.port = 0
+          remote.artery {
+            canonical {
+              hostname = 127.0.0.1
+              port = 0
+            }
+          }
+        }
+        """)
+
+      val system1 = ActorSystem(Behaviors.empty[String], "sys1", config)
+      val system2 = ActorSystem(Behaviors.empty[String], "sys2", config)
+
+      try {
+
+        val remoteRefStr = ActorRefResolver(system1).toSerializationFormat(ActorRef.ignore[String])
+        remoteRefStr should (startWith("akka://sys1@127.0.0.1") and endWith("/system/ignore"))
+
+        val ref: ActorRef[String] = ActorRefResolver(system2).resolveActorRef[String](remoteRefStr)
+        ref.path.toString should (startWith("akka://sys1@127.0.0.1") and endWith("/system/ignore"))
```

Comment:
I would prefer to have the path set to `sys2`. As it is now, the reply will be sent back to system1 and will be ignored there. It will be better if we can simply ignore it locally.

Reply:
I managed to solve this. See second commit.